geode-core/src/distributedTest/java/org/apache/geode/cache30/PRBucketSynchronizationDUnitTest.java

```diff
@@ -211,44 +210,22 @@ private void verifySynchronized(VM vm, final InternalDistributedMember crashedMe
       PartitionedRegion pr = (PartitionedRegion) testRegion;
       final BucketRegion bucket = pr.getDataStore().getLocalBucketById(0);
 
-      Wait.waitForCriterion(new WaitCriterion() {
-        String waitingFor = "primary is still in membership view: " + crashedMember;
-        boolean dumped = false;
-
-        @Override
-        public boolean done() {
-          if (testRegion.getCache().getDistributionManager().isCurrentMember(crashedMember)) {
-            return false;
-          }
-          if (!testRegion.containsKey("Object3")) {
-            waitingFor = "entry for Object3 not found";
-            return false;
-          }
-          RegionEntry re = bucket.getRegionMap().getEntry("Object5");
-          if (re == null) {
-            if (!dumped) {
-              dumped = true;
-              bucket.dumpBackingMap();
-            }
-            waitingFor = "entry for Object5 not found";
-            return false;
-          }
-          if (!re.isTombstone()) {
-            if (!dumped) {
-              dumped = true;
-              bucket.dumpBackingMap();
-            }
-            waitingFor = "Object5 is not a tombstone but should be: " + re;
-            return false;
-          }
-          return true;
+      Awaitility.await().until(() -> {
```

Comment:
Are these changes needed because of the change to using a system timer for scheduling the sync? If so do these tests now take longer to run as they wait on the system timer?

Reply:
These changes were just to replace a WaitCriterion with Awaitility and to remove debug dumping of the RegionMap